src/test/java/io/vertx/ext/mail/mailencoder/MailEncoderTest.java

```diff
@@ -206,10 +206,10 @@ public void testToManyEncoded() throws Exception {
     String mime = new MailEncoder(message, HOSTNAME).encode();
     assertThat(mime, containsString("To: user0@example.com (=?UTF-8?Q?=C3=84a?=),user1@example.com\n"
       + " (=?UTF-8?Q?=C3=84a?=),user2@example.com (=?UTF-8?Q?=C3=84a?=),\n"
-      + " user3@example.com (=?UTF-8?Q?=C3=84a?=),user4@example.com (=?UTF-8?Q?=C3?=\n"
-      + " =?UTF-8?Q?=84a?=),user5@example.com (=?UTF-8?Q?=C3=84a?=),\n"
-      + " user6@example.com (=?UTF-8?Q?=C3=84a?=),user7@example.com (=?UTF-8?Q?=C3?=\n"
-      + " =?UTF-8?Q?=84a?=),user8@example.com (=?UTF-8?Q?=C3=84a?=),\n"
+      + " user3@example.com (=?UTF-8?Q?=C3=84a?=),user4@example.com (=?UTF-8?Q??=\n"
```

Comment:
As you can see, `=C3=84` should not fold into the next line.
But also note that we may have "**??=**\n" at the end of the line, but I think is allowed.